### DISCUSSION TITLE
Default Remix Vercel boilerplate does not work?

### DISCUSSION THREAD

**Discussion by vedantroy** [remix-run/remix#3009] [Q&A]
I created a project with `npx create-remix@latest` and selected vercel for deployment. I then deployed the site using `npx vercel`. However, I'm getting a 404 whenever I go to the project page: https://new-vercel-pi.vercel.app/.

I assume that the default template should work, is there something obvious I'm missing here?

**Comment by nickwanhere** [comment]
Mine was working few days ago, now it is showing 404.

I checked the working one log to the failed one. The working one exported the api/index while the other didn't. 

Something funny on vercel.

**Reply by nickwanhere** [reply]
I end up moved to netlify for the same codebase.
